source/archive/tar.d

```diff
@@ -560,7 +561,7 @@ public class TarPolicy
                 header.ownerId = rightJustify(intToOctalStr(0), 7) ~ "\0";
                 header.groupId = rightJustify(intToOctalStr(0), 7) ~ "\0";
                 header.size = rightJustify(intToOctalStr(0), 11) ~ " ";
-                header.modificationTime = rightJustify(longToOctalStr(directory.modificationTime), 11) ~ " ";
```

Comment:
you removed the 11 padding here, which works for times after 2004, but what about older ones that aren't 11 octal digits? I don't really know how the tar format works internally, but I assume the 11 was here for a reason for small dates before.

Reply:
It's 11 chars, but the padding is usually 0 with newer program (the field is nullified earlier in the code).
So what this does essentially is replace space padding with `\0` padding.